src/index.js

```diff
@@ -9,13 +9,21 @@ export default function factory({
   onError = onerror,
   onNoMatch = onerror.bind(null, { status: 404, message: "not found" }),
   attachParams = false,
+  disableResponseWait = false,
 } = {}) {
-  function nc(req, res) {
+  async function nc(req, res) {
+    let closeP;
```

Comment:
Minor thing, but this could be called `finishP` now. Would be a bit more readable.